.github/workflows/test_simple.bat

```diff
@@ -0,0 +1,3 @@
+set grass=%1\grass79
```

Comment:
From the commits, I see you had an issue calling just the executable on path, but you could potentially pass a path to the actual executable which seems more aligned with what a user would do, or not? This would decrease the number of places where we need to change grass79 to something else when the time comes (unless you just create also another one which would be just grass).

Reply:
Agreed